src/main/scala/diplomacy/Nodes.scala

```diff
@@ -411,11 +411,11 @@ sealed abstract class MixedNode[DI, UI, EI, BI <: Data, DO, UO, EO, BO <: Data](
   private var bundlesSafeNow = false
   // Accessors to the result of negotiation to be used in LazyModuleImp:
   def out: Seq[(BO, EO)] = {
-    require(bundlesSafeNow, s"${name}.out should only be called from the context of its module implementation")
+    require(bundlesSafeNow, s"${name}.out should only be called from the context of its LazyModule(${scope.get.name}) implementation, currently (${chisel3.Module.currentModule.get.name})")
```

Comment:
Are you sure `chisel3.Module.currentModule.get` is always defined when this error case can arise?
```
class Parent extends SimpleLazyModule {
   val child = LazyModule(new SimpleLazyModule { val node = ... })
   val error = child.node.out.head
}
```
Referencing `child.node.out` in Parent's LazyModule scope should trip this requirement, but I don't think `chisel3.Module.currentModule` is defined, or if it is, it will be the very top-level testharness Module or something confusing like that?

Reply:
sorry my bad, I'm looking into it.

Reply:
`scope.get.name` should always being defined, and I use `LazyModule.scope` to get current scope of LazyModule, maybe it will fall back to `None`, but generally solve this problem.